### DISCUSSION TITLE
[Feature Request]: Monthly/Weekly stats

### DISCUSSION THREAD

**Discussion by kkkbe** [ciderapp/Cider#1009] [Feature Request]
### The feature you would like

Hi! I have seen several Spotify apps, that gives user their monthly, weekly listening stats, but never any that works with Apple Music. I love Cider. And I believe that it wouldn't be very hard to add this feature. If I understand how things work, then this feature would work by comparing last week's (or month's) stats with new ones (since Apple updates them once a week). Of course, I might be wrong.  

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by quacksire** [comment]
The closest thing we have right now is the Replay page and integration with last.fm

**Comment by kkkbe** [comment]
I undestand that, but I believe that it might be cool to have this feature in-app.

**Comment by xx88xx** [comment]
+1 would love this
